tensorflow-base-cpu-linwin/meta.yaml

```diff
@@ -110,8 +106,8 @@ requirements:
     - sqlite
     - snappy
     - typing_extensions >=3.7.4
-    - tensorboard >=2.6.0,<2.7
-    - tensorflow-estimator >=2.6.0,<2.7
+    - tensorboard >=2.8,<2.9
+    - tensorflow-estimator >=2.8,<2.9
   run:
     - python
     - absl-py >=0.10.0
```

Comment:
flatbuffers >= 1.12 is required, see https://github.com/tensorflow/tensorflow/blob/2ea19cbb575d076b4f521d3603211c8316ad5f8f/tensorflow/tools/pip_package/setup.py#L76

Reply:
`google_pasta >=0.1.1` is required, see https://github.com/tensorflow/tensorflow/blob/2ea19cbb575d076b4f521d3603211c8316ad5f8f/tensorflow/tools/pip_package/setup.py#L78

Reply:
`h5py >= 2.9.0`, https://github.com/tensorflow/tensorflow/blob/2ea19cbb575d076b4f521d3603211c8316ad5f8f/tensorflow/tools/pip_package/setup.py#L79